circle.yml

```diff
@@ -47,7 +47,20 @@ jobs:
     working_directory: ~/addons-server
     machine: true
     steps:
-      - checkout
+      - run:
+          name: Checkout git repository
+          command: |
+            git clone --single-branch --depth 1 --branch $CIRCLE_BRANCH https://github.com/$CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME.git ~/addons-server
+
+            if [ -n "$CIRCLE_TAG" ]
+            then
+              git reset --hard "$CIRCLE_SHA1"
+              git checkout -q "$CIRCLE_TAG"
+            elif [ -n "$CIRCLE_BRANCH" ]
+            then
+              git reset --hard "$CIRCLE_SHA1"
+              git checkout -q -B "$CIRCLE_BRANCH"
+            fi
```

Comment:
This is needed to add git clone --depth 1 support. Apparently circleci doesn't support it ootb, only version 1 did :(

Reply:
mmm I see that the `build-release` job doesn't have this... Worth checking with ops if we could add that there as well to speed up our build creation...

Reply:
We should, I opened https://github.com/mozilla/addons-server/issues/7488 to track this. Shouldn't be much work, I'd rather do it in a separate PR though.